doc/BENCHMARKING.md

```diff
@@ -32,12 +32,12 @@ You can use the following steps.
                 --fuse-version 2 \
                 --with-fio --with-libunwind
 
-2. Set environment variables related to the benchmark. There are four required environment variables you need to set in order to run the benchmark.
+2. Set environment variables related to the benchmark. There are four required environment variables you need to set in order to run the benchmark. These variables can be set to any name, but be consistent with it in subsequent steps.
```

Comment:
Should we change these steps to first create the files needed and then populate the env variables?

Also, on line 45, there is a `DOC-EXAMPLE-BUCKET` which is undefined. Please add that to the set of variables to set.

Reply:
Yes. That seems more reasonable. Making the change. Thanks